desktop/cmp/dimensionchooser/DimensionChooser.js

```diff
@@ -41,6 +41,7 @@ export const [DimensionChooser, dimensionChooser] = hoistCmp.withFactory({
         emptyText = 'Ungrouped',
         popoverWidth = 250,
         popoverTitle = 'Group By',
+        popoverPosition = 'bottom',
```

Comment:
should the default be auto?

Reply:
john points out that auto actually has very bad behavior, in that the popover can change size when it changes state and move!  Let's go for the non-breaking change right now that leaves the default at 'bottom'